### PR TITLE
Updated config.toml for 1.32 release

### PR DESCRIPTION
**Pull request by chanieljdan**
Updating hugo.toml for the 1.32 release.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm

File: hugo.toml [hunk 1]
```diff
@@ -185,11 +185,17 @@ js = [
...
+version = "v1.32"
+githubbranch = "v1.32.0"
 docsbranch = "main"
 url = "https://kubernetes.io"
 
+[[params.versions]]
+version = "v1.31"
+githubbranch = "v1.31.0"
+docsbranch = "release-1.31"
+url = "https://v1.31.docs.kubernetes.io"
```
Comment (tengqm) [inline hugo.toml:197]:
```suggestion
url = "https://v1-31.docs.kubernetes.io"
```

Comment (chanieljdan) [reply hugo.toml]: Thanks @tengqm, fixed!

**Comment by sftim** [comment]
Ah yep.

/lgtm

**Pull request merged**